test/hotspot/jtreg/compiler/loopopts/Test8211698.java

```diff
@@ -54,4 +54,3 @@ public void test() {
         }
     }
 }
-
```

Comment:
You can leave this empty line in at the end of the file.

Reply:
done